generator/app/VerificationBench.hs

```diff
@@ -262,6 +262,7 @@ main = do
                     when (errno > 0) $ do
                         logError $ sformat ("Verification/Application errors ("%shown%"):") errno
                         traverse_ (logError . show) errs
+    removeAllHandlers lh
```

Comment:
This is fine for now and to be cherrypicked back to the release/2.0.0 branch, but after that we may want to replace these `setupLogging` / `removeAllHanlders` call pairs with a function `withLoggingSetup` to properly bracket the acquire/release process.